packages/form/src/Feedback.js

```diff
@@ -5,7 +5,17 @@ import { ErrorMessage } from 'formik';
 
 const Feedback = ({ name, ...rest }) => {
   const feedbackId = `${name}-feedback`.toLowerCase();
-  return <ErrorMessage id={feedbackId} component={FormFeedback} name={name} {...rest} />;
+  const AvFeedback = ({ children, ...rest }) => (
+    <FormFeedback {...rest}>
+      <i className="icon icon-attention" aria-label="Error" />
+      {children}
+    </FormFeedback>
+  );
+
+  AvFeedback.propTypes = {
```

Comment:
We should move AvFeedback out of Feedback so we can do the proptypes outside of it as well, and it just looks nicer when all the proptypes are together.